### ISSUE TITLE
Should this repo be archived as the library is obsolete?

### ISSUE THREAD

**Issue by cmorganBE** [littlevgl/arduino#63]
I'm learning about lvgl and was looking through folders in this repo and then found the top level 'obsolete' message. Should this repo be archived to avoid confusing people looking around that don't know what they are looking at/for?

**Comment by kisvegabor** [comment]
That's true. I wanted to wait for a little while the lvgl repo becomes more stable for Ardunio.
 
I'm just archiving it.
